docs/templates/config.rst.j2

```diff
@@ -40,6 +40,43 @@ Ansible will process the above list and use the first file found, all others are
         inventory = /etc/ansible/hosts  ; This points to the file that lists your hosts
 
 
+.. _cfg_in_world_writable_dir:
+
+Configuration file when the current directory is world writable
```

Comment:
I'd change the title to tell folks why they (should) care about this:
"Mitigating security risks of using local ansible.cfg"
and then reorganize the paragraph so we state the issue, the easy fix, the warning, and the "if you must" part. I can do that bit . . .